examples/dt/dcn/service-values.yaml

```diff
@@ -0,0 +1,246 @@
+# local-config: referenced, but not emitted by kustomize
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: service-values
+  annotations:
+    config.kubernetes.io/local-config: "true"
+data:
+  preserveJobs: false
+  cinderAPI:
+    replicas: 3
```

Comment:
We need to define the default AZ so some Volume resources such as Volumes groups created in tempest tests have a proper AZ to be created at, something like:
```
cinderAPI:
    replicas: 3
    customServiceConfig: |
      [DEFAULT]
        default_availability_zone = ceph
```

Reply:
Done